lib/registrant.rb

```diff
@@ -0,0 +1,20 @@
+class Registrant
+    attr_reader :name,
+                :age,
+                :license_data
+
+    def initialize(name, age, permit = false)
+        @name = name
+        @age = age
+        @permit = permit
+        @license_data = {:written=>false, :license=>false, :renewed=>false}
+    end
+
+    def permit?
+        @permit == true
```

Comment:
Since `@permit` is storing a boolean value, you do not need to do a comparison operator to another boolean. The contents of the `permit?` method could just be `@permit`